src/instantx.cpp

```diff
@@ -117,8 +117,8 @@ bool CInstantSend::ProcessTxLockRequest(const CTxLockRequest& txLockRequest, CCo
                 if(hash != txLockRequest.GetHash()) {
                     LogPrint("instantsend", "CInstantSend::ProcessTxLockRequest -- Double spend attempt! %s\n", txin.prevout.ToStringShort());
                     // do not fail here, let it go and see which one will get the votes to be locked
-                    // NOTIFY SCRIPTS + ZMQ
-                    GetMainSignals().NotifyInstantSendDoubleSpendAttempt(txin.prevout);
+                    // NOTIFY ZMQ
+                    GetMainSignals().NotifyInstantSendDoubleSpendAttempt(txHash, hash);
```

Comment:
Should pass txes here, not hashes, just like you described in your previous comment re the issue https://github.com/dashpay/dash/pull/2262#discussion_r215264029. There is no use of "raw" hashes, the whole point of "raw" messages is to provide the _object_ that triggered the notification.

Reply:
I tried to do such, but I could not find a way to get the CTransaction objects based on what is there